app/build.gradle

```diff
@@ -102,3 +103,38 @@ dependencies {
     releaseCompile 'com.squareup.leakcanary:leakcanary-android-no-op:1.3.1' // or 1.4-beta1
     testCompile 'com.squareup.leakcanary:leakcanary-android-no-op:1.3.1' // or 1.4-beta1
 }
+
+jacoco {
```

Comment:
all this is not needed btw, 

gradle has jacoco support.

I'll merge this nevertheless,

I'll do the unit test setup myself later.